### ISSUE TITLE
Incoherent doc comments

### ISSUE THREAD

**Issue by shika-blyat** [Julien00859/regexp#1]
This [comment](https://github.com/Julien00859/regexp/blob/master/regexp/pattern.py#L27) states that `a?` is avaible as an alias of  `(a|epsilon)` and this [one](https://github.com/Julien00859/regexp/blob/master/regexp/pattern.py#L34) states it is not.

**Comment by Julien00859** [comment]
Indeed, that's some old docstrings I forgot to update, thank you :) I'll be happy yo merge a PR that fixes the problem, don't forget to regenerate the doc.

**Comment by shika-blyat** [comment]
> don't forget to regenerate the doc

Which part of the doc should be updated ? 🤔

**Comment by Julien00859** [comment]
The `?` is a valid alias since 900d60d0698
